docs/components/tasklist/userguide/using-filters.md

```diff
@@ -0,0 +1,87 @@
+---
+id: using-filters
+title: Using filters
+description: "Find relevant tasks in Tasklist"
+---
+
+When you open the **Tasks** page, you will see the list of all tasks. This is a starting point for managing your workload.
+
+![tasklist-all-tasks](img/task-filters/tasklist-all-tasks.jpg "All open tasks")
+
+## Apply a filter
+
+As you browse through the list, you can utilize filters to locate specific tasks and organize your workload.
+
+### Select a filter
+
+You can select one of the default filters or [create a new filter](#create-new-filter).
+
+Default filters:
+
+- All open tasks
+- Assigned to me
+- Unassigned
+- Completed tasks
+
+![tasklist-default-filters](img/task-filters/tasklist-default-filters.jpg "List of the default filters")
+
+## Create new filter
+
+To create a new filter, click to apply a new filter. This action will open a filter dialog.
+In the dialog, you can define various rules based on task attributes. The supported attributes are:
+
+- Assignees and candidate groups
+- Status
+- Processes tasks belong to
+- Dates (due date and follow up date)
+- Task ID
+- Task variables
+
+![tasklist-filter-dialog-with-advanced-options](img/task-filters/tasklist-filter-dialog-with-advanced-options.jpg "Available filter attributes")
+
+After defining the rules, click **Apply** to apply the filter. The system will then filter the tasks according to your criteria, displaying only the relevant tasks.
+
+## Save filter for future use
+
+If you need to use the filter again, you can save it:
+
+1. Click **Save** filter.
```

Comment:
Last question 👍  Is the UI **Save** or **Save filter**?

Reply:
@christinaausley The CTA is called **Save**. I pushed an update for this.